acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/OptimisticSafeSlotsSyncPostMergeAcceptanceTest.java

```diff
@@ -0,0 +1,115 @@
+import tech.pegasys.teku.infrastructure.time.SystemTimeProvider;
+import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.test.acceptance.dsl.AcceptanceTestBase;
+import tech.pegasys.teku.test.acceptance.dsl.BesuNode;
+import tech.pegasys.teku.test.acceptance.dsl.TekuNode;
+import tech.pegasys.teku.test.acceptance.dsl.tools.deposits.ValidatorKeystores;
+
+public class OptimisticSafeSlotsSyncPostMergeAcceptanceTest extends AcceptanceTestBase {
+  private static final String NETWORK_NAME = "less-swift";
+  private static final Integer SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY = 4;
+
+  private final SystemTimeProvider timeProvider = new SystemTimeProvider();
+  private BesuNode eth1Node1;
+  private BesuNode eth1Node2;
+  private TekuNode tekuNode1;
+  private TekuNode tekuNode2;
+  private String eth1Node1Enode;
+
+  @BeforeEach
+  void setup() throws Exception {
+    final int genesisTime = timeProvider.getTimeInSeconds().plus(10).intValue();
+    eth1Node1 =
+        createBesuNode(
+            config ->
+                config
+                    .withMiningEnabled()
+                    .withMergeSupport(true)
+                    .withP2pEnabled()
+                    .withGenesisFile("besu/preMergeGenesis.json"));
+    eth1Node1.start();
+    eth1Node1Enode = eth1Node1.fetchEnodeUrl();
+    eth1Node2 =
+        createBesuNode(
+            config ->
+                config
+                    .withMergeSupport(true)
+                    .withP2pEnabled()
+                    .withGenesisFile("besu/preMergeGenesis.json")
+                    .withStaticNodes(eth1Node1Enode));
+    eth1Node2.start();
+
+    final int totalValidators = 4;
+    final ValidatorKeystores validatorKeystores =
+        createTekuDepositSender(NETWORK_NAME).sendValidatorDeposits(eth1Node1, totalValidators);
+    tekuNode1 =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, eth1Node1, genesisTime)
+                    .withValidatorKeystores(validatorKeystores)
+                    .withValidatorProposerDefaultFeeRecipient(
+                        "0xFE3B557E8Fb62b89F4916B721be55cEb828dBd73"));
+    tekuNode1.start();
+    tekuNode2 =
+        createTekuNode(
+            config ->
+                configureTekuNode(config, eth1Node2, genesisTime)
+                    .withInteropValidators(0, 0)
+                    .withPeers(tekuNode1)
+                    .withSafeSlotsToImportOptimistically(SAFE_SLOTS_TO_IMPORT_OPTIMISTICALLY));
+    tekuNode2.start();
+  }
+
+  @Test
+  void shouldPassMergeOptimisticallyAndBeginFinalizationAfterSafeSlotsToImport() throws Exception {
+    tekuNode2.waitForGenesis();
+    // Disconnect eth1Node2 from eth1Node1 before the merge
+    assertThat(eth1Node2.removePeer(eth1Node1Enode)).isTrue();
+    tekuNode1.waitForLogMessageContaining("MERGE is completed");
+
+    tekuNode2.waitForLogMessageContaining("optimistic sync");
```

Comment:
There could be a lot of messages that contain optimistic sync.  We probably need to be more specific about what we're looking for.  Likely the best thing is to retrieve the head block and check that the `execution_optimistic` field is true for it which indicates it was imported optimistically.  That would also only be true once Teku had sync'd past the actual merge block so could probably replace all the "MEREGE completed" log message as well.